content/en/users/tutorials/filetransfer-gridstorage/_index.md

```diff
@@ -8,4 +8,229 @@ description: >
+
+The last option that we need to use is specify the VO that we want to use for
+the proxy being created. In the following example the "dteam" VO has been used:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+[ac@localhost ~]$ voms-proxy-init --voms dteam
+Your identity: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+Creating temporary proxy ................................................................... Done
+Contacting  voms2.hellasgrid.gr:15004 [/C=GR/O=HellasGrid/OU=hellasgrid.gr/CN=voms2.hellasgrid.gr] "dteam" Done
+Creating proxy .............................................................................. Done
+
+Your proxy is valid until Wed Sep  8 04:37:07 2021
+```
+
+<!-- markdownlint-enable line-length -->
+
+### Find the storage
+
+In general it should be already known the source and destination storage for a
+specific project however to find the available options we can use for a
+transfer, source and destination, we can use the
+[VAPOR service](https://operations-portal.egi.eu/vapor/resources/GL2ResVO).
+
+![VAPOR main page](VAPOR-home.png)
+
+Once the page is loaded on the left column it is possible to filter by VO or
+scroll the list and click the desired VO as show in the following picture:
+
+![VAPOR VO detail](VAPOR-VO_detail.png)
+
+Once selected we can see all the resources associated with the spcific VO. In
+particular in this case we're interested in the information on the status,
+capacity, type of storage, etc. Following is a screenshot of the visualization
+of the list of storage available to "dteam"
+
+![VAPOR storage page](VAPOR-storage.png)
+
+###Start a transfer
+
+Once we have identified the surce and destination storage needed for the
+transfer we can proceed with the transfer between the two endpoints. To do that
+we can use a command of this type:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ fts-transfer-submit -s https://fts3-public.cern.ch:8446/ --source https://dc2-grid-64.brunel.ac.uk/dpm/brunel.ac.uk/home/dteam/00 --destination https://golias100.farm.particle.cz/dpm/farm.particle.cz/home/dteam/00
+d7920806-10b1-11ec-8218-fa163ecee758
+```
+
+<!-- markdownlint-enable line-length -->
+
+Which return the jobid correspondedent to the transfer that we started. To check
+the status of the transfer we can use the returned jobid and use the following
+command:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ fts-transfer-status -s https://fts3-public.cern.ch:8446/ d7920806-10b1-11ec-8218-fa163ecee758
+ACTIVE
+```
+
+<!-- markdownlint-enable line-length -->
+
+which specify the server that control the transfer, the source and the transfer
+itself.
+
+## WebFTS
+
+### Access the WebFTS interface
+
+The WebFTS is accessible at this URL, [WebFTS](https://webfts.cern.ch/)
+Similarly to what has been done from the command line interface we need to
+provide our private key for delegation of the credential. To do that we use the
```

Comment:
Maybe it could be worth to add a warning about the fact that the decrypted private key should be treated carefully.

Reply:
**Warning**: please be careful and avoid sharing this information with any third party or saving this information in plain text. WebFTS uses the key to acquire a proxy certificate on your behalf as described previously and doesn't store it.

Reply:
Looks good

Reply:
does not